docassemble/ALToolbox/al_income.py

```diff
@@ -441,19 +513,19 @@ def owners(self, source=None):
 
 class ALVehicle(ALAsset):
     """
-    One vehicle with its properties. Extends ALAsset. Vehicles have a
-        .year_make_model() method.
+    An ALAsset with special attributes that help representing a vehicle.
+
+    Vehicles have a .year_make_model() method which facilitates
+    listing vehicles on many financial statement forms.
 
     Attributes:
-    .year {str} The year of the vehicle.
-    .make {str} The make of the vehicle.
-    .model {str} The model of the vehicle.
-    .market_value {num} Market value of an asset.
-    .balance {num} Balance of an asset.
-    .times_per_year {str | num} Represents a number of the annual frequency of
-        the value. E.g. 12 for a monthly value.
-    .value {num} (Optional) Represents an amount of money accumulated by this
-        asset over its `times_per_year`. Default is 0.
+    .year {str} The model year of the vehicle, like 2022
+    .make {str} The make of the vehicle, like "Honda"
+    .model {str} The model of the vehicle, like "Accord"
+    .market_value {int | float} Market value of an asset.
+    .balance {int | float} Balance of an asset.
```

Comment:
Considering your description of a balance further up, does a vehicle need a balance?